docs/customresourcestate-metrics.md

```diff
@@ -287,7 +289,7 @@ spec:
 
 Produces:
 ```prometheus
-myteam_foos_uptime 43.21
+myteam_foos_uptime{group="myteam.io", kind="Foo", version="v1"} 43.21
```

Comment:
```suggestion
uptime{group="myteam.io", kind="Foo", version="v1"} 43.21
```

Reply:
Would you mind changing the prefix to "kube_crd" as the other comment suggested?

Reply:
https://github.com/kubernetes/kube-state-metrics/pull/1850#discussion_r991572154

Reply:
This metric would be unchanged as the configuration specified a `metricNamePrefix: myteam_foos`.

https://github.com/kubernetes/kube-state-metrics/blob/633cbf49d0ecf607c716860b21e03c4afd596b8c/docs/customresourcestate-metrics.md?plain=1#L279-L293